docs/en/status-communities/delegate-admin-functions-to-your-community.md

```diff
@@ -0,0 +1,60 @@
+---
+id: 543
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Delegate admin functions to your community
```

Comment:
```suggestion
# Delegate admin functions to your community member
```

Reply:
Setting "Delegate admin functions to community members" title.